cypress/fixtures/user.json

```diff
@@ -1,11 +1,15 @@
 {
-  "email":"testone@@.co",
-  "emailone":"trainingcypress@wolox.com.co",
-  "First_name": "Sandra",
-  "Last_name": "Marin",
-  "Email":"sandraprueba@gmail.com",
-  "Password": "Wolox1189",
-  "email_one":"marcela.soto@wolox.co",
-  "password":"wolox1189"
+  "email" : "testone@@.co",
+  "emailone" : "trainingcypress@wolox.com.co",
+  "First_name" : "Sandra",
+  "Last_name" : "Marin",
+  "Email" : "sandraprueba@gmail.com",
+  "Password" : "Wolox1189",
+  "email_one" : "marcela.soto@wolox.co",
+  "password1" : "wolox1189",
+  "email_two" : "conduit1189@wolox.co",
+  "email_i" : "kkkkkk",
+  "username" : "Chela",
+  "pwd" : "123"
```

Comment:
What do you think about splitting this into multiple fixtures for each case or having nested objects?
Like:
```
{
  "completeUser": {
    "email" : "testone@@.co",
    "username" : "Chela",
    "password" : "123"
  },
  "userWithoutName": {
    "email" : "testone@@.co",
    "username" : "",
    "password" : "123"
  }
}
```
And so on

I think it'd be easier to mantain.